ui/src/components/Notifications.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Txt, Alert } from 'rendition';
+
+export const Notifications = ({
+  hasAvailableNetworks,
+  isConnecting,
+  error
+}: {
+  hasAvailableNetworks: boolean;
+  isConnecting: boolean;
+  error: string;
+}) => {
+  return (
+    <>
+      {isConnecting && (
+        <Alert info>
+          <Txt.span>Applying changes... </Txt.span>
+          <Txt.span>
+            Your device will soon be online. If connection is unsuccessful, the
+            Access Point will be back up in a few minutes, and reloading this
+            page will allow you to try again.
+          </Txt.span>
+        </Alert>
+      )}
+      {!hasAvailableNetworks && (
+        <Alert warning>
+          <Txt.span>No wifi networks available. </Txt.span>
```

Comment:
There is a trailing space here:
```suggestion
          <Txt.span>No wifi networks available.</Txt.span>
```

Reply:
That's intentional and shown in the UI, but I replaced it with `&nbsp;` for clarity.